src/shared-components/NotificationToast/NotificationToast.spec.tsx

```
@@ -0,0 +1,169 @@
+  NotificationToast,
+  NotificationToastContainer,
+} from './NotificationToast';
+
+// Clear mocks after each test for isolation across all describe blocks
+afterEach(() => {
+  vi.clearAllMocks();
+});
+
+describe('NotificationToast', () => {
+
+  it('calls toast.success with defaults for a string message', () => {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)